op-deployer/pkg/deployer/upgrade/flags.go

```diff
@@ -24,11 +25,19 @@ var Commands = cli.Commands{
 		Usage: "upgrades a chain to version v2.0.0",
 		Flags: append([]cli.Flag{
 			deployer.L1RPCURLFlag,
-			deployer.DeploymentTargetFlag,
-			deployer.PrivateKeyFlag,
```

Comment:
Just confirming we're intending to simplify the functionality here and always just execute this as a calldata target.

Reply:
yep this is correct